dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/DataOkHttpUploader.kt

```diff
@@ -16,7 +16,8 @@ import okhttp3.RequestBody
 
 internal abstract class DataOkHttpUploader(
     private var url: String,
-    private val client: OkHttpClient
+    private val client: OkHttpClient,
+    internal val contentType: String = CONTENT_TYPE_JSON
```

Comment:
I think I've seen this also in your latest PR right ?

Reply:
Yes for the RUM data, i'll rebase it afterwards ;)